validator/lib/validateTileset.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+var Promise = require('bluebird');
+
+module.exports = validateTileset;
+
+/**
+ * Checks whether the geometricError of each tile is <= the geometric error of its parent tile.
```

Comment:
Tweak this description so it is more generic, since `validateTileset` will do more than just check geometric error.